### DISCUSSION TITLE
coupled Evolution of Deformation, Pore Fluid Pressure

### DISCUSSION THREAD

**Discussion by ch9411** [GEOS-DEV/GEOS#2819] [Q&A]
Is it possible to use underworld to build numerical sandbox modeling and research on the coupled Evolution of Deformation, Pore Fluid Pressure, and Fluid Flow in the thrust-fold mountain belt ？

**Comment by paveltomin** [comment]
that would most probably require large deformation models, not sure GEOS is the best tool for that
